nima/tests/integration/webserver/webserver/src/test/java/io/helidon/nima/tests/integration/server/RoutingTestBase.java

```diff
@@ -0,0 +1,151 @@
+    }
+
+    @Test
+    void testHttpShortcutMethodsWithoutPathPattern() {
+        List<Map<String, Http1ClientRequest>> requests = Arrays.asList(Map.of("get_catchall", client.get("/get_catchall")),
+                                                                       Map.of("post_catchall", client.post("/post_catchall")),
+                                                                       Map.of("put_catchall", client.put("/put_catchall")),
+                                                                       Map.of("delete_catchall",
+                                                                              client.delete("/delete_catchall")),
+                                                                       Map.of("head_catchall", client.head("/head_catchall")),
+                                                                       Map.of("options_catchall",
+                                                                              client.options("/options_catchall")),
+                                                                       Map.of("trace_catchall", client.trace("/trace_catchall")),
+                                                                       Map.of("patch_catchall", client.patch("/patch_catchall")));
+
+        validateHttpShortCutMethods(requests);
+    }
+
+    @Test
+    void testHttpShortcutMethodsWithPatchMatcher() {
+        List<Map<String, Http1ClientRequest>> requests = Arrays.asList(Map.of("wildcard_test1", client.get("/wildcard_any")),
+                                                                       Map.of("wildcard_test2", client.post("/wildcard/any")));
+        validateHttpShortCutMethods(requests);
+    }
+
+    @Test
+    void testHttpShortcutMethodsWithMultiHandlers() {
+        List<Map<String, Http1ClientRequest>> requests = Arrays.asList(Map.of("get_multi", client.get("/get_multi")),
+                                                                       Map.of("post_multi", client.post("/post_multi")),
+                                                                       Map.of("put_multi", client.put("/put_multi")),
+                                                                       Map.of("delete_multi",
+                                                                              client.delete("/delete_multi")),
+                                                                       Map.of("head_multi", client.head("/head_multi")),
+                                                                       Map.of("options_multi",
+                                                                              client.options("/options_multi")),
+                                                                       Map.of("trace_multi", client.trace("/trace_multi")),
+                                                                       Map.of("patch_multi", client.patch("/patch_multi")));
+
+        validateHttpShortCutMethods(requests, true);
+    }
```

Comment:
Parametrized test could be more elegant

Reply:
looking into this

Reply:
Converted all tests that iterate into ParameterizedTest as suggested.